### DISCUSSION TITLE
Custom Reports - Dependent Cascading Lists

### DISCUSSION THREAD

**Discussion by alexgarix** [shridhar-tl/jira-assistant#332] [Q&A]
Hi.

Thanks a lot for share this tool with all us. It's great!

I have a problem when I'm getting information of a custom field created as dependent cascading list, let's call it "Item" field. It has two select list, displayed options of 2nd one are dependent on the selection of 1st one. As example:

- If I select "A" in 1st list , then I have "2,3,4,5" in 2nd list
- If I select "B" in 1st list , then I have "1,6,8" in 2nd list
- If I select "C" in 1st list , then I have "7" in 2nd list

I add this custom field in "Display settings". I see field type "com.atlassian.jira.plugin.system.customfieldtypes:cascadingselect", but when I run report view I'm only able to see content of 1st select list ("A", "B" or "C"), I never get the content of 2nd one like in jira search filters ("A - 2", "A - 5", "B - 1", "C - 7").

Is there any way to get the complete information of this field?

Thanks for your time and I apologyse if my English is not good enought.

**Comment by shridhar-tl** [comment]
Hi @alexgarix, Thank you for reporting this issue.

I will look into this issue by this week end and will try to solve it.

Additionally, I would recommend using Pivot report instead, which has almost all the capability of custom report, but has lot more flexibility in terms of grouping and filtering.

I am not stating this field may work in pivot report. Even pivot report could have same issue which I will try to fix by this week end. However, I am thinking about replacing custom report with this new Pivot Report as this is more capable and have more features lined up.

Currently Pivot report is only available in Web version. By this week end will try to publish it for extension users as well.

You can read more about it here: https://jiraassistant.com/features/reports/pivot/

**Reply by alexgarix** [reply]
Hi.

Thanks for you fast answer! Unfortunately I can't check it with Web version, my JIRA instance is not accessible from outside and I guess it will not link with web version.

About fix, for me it's not urgent so take your time ;) I can get this information by other longer way that I use to.


BR

**Reply by shridhar-tl** [reply]
Hi @alexgarix, Doesn't matter. Even web version is similar to extension. It doesn't have a backend server as such which access your Jira.

Even with Web version still your browser only connects with Jira. Not our backend server. So if you have access to internet and if were able to access the following Url, it works even if your Jira instance is internal to your organization and not exposed in internet.

So feel free to just give a try hitting the following Url and it should automatically load without needing for any configuration (as extension is already installed and configured): https://app.jiraassistant.com/?authType=1

Of course, its up to you. Was just trying to clarify that Web version also doesn't require your Jira server being exposed in internet. It works with server / self hosted version of Jira as well.

**Reply by alexgarix** [reply]
Hi @shridhar-tl 

I had chance to run Web version with link that you provided and all looks working fine (at least same options that I usually need), but Pivot Report is not showing me any field after set raw JQL query (project = "ABC" and created >="2023-10-01"), it just shows "Drag and drop required fields from list to be shown in report". Same query works me properly in Custom Reports.

Let me know if you need more information from my side.


BR
